packages/frontend/src/Build.svelte

```diff
@@ -881,6 +913,56 @@ $: if (availableArchitectures) {
                     id="buildConfigKernelArguments"
                     placeholder="Kernel arguments (ex. quiet)"
                     class="w-full" />
+
+                  <!-- The below section is related to anaconda-iso, only show if buildType has 'anaconda-iso' in it -->
```

Comment:
Is this comment still relevant? I don't understand how it is implemented

Reply:
Woops! Leftover comment should of been removed. Good catch.